lib/ecl/ecl_file_kw.c

```diff
@@ -422,3 +424,12 @@ ecl_file_kw_type * ecl_file_kw_fread_alloc( FILE * stream ) {
 
   return file_kw;
 }
+
+
+void ecl_file_kw_increment_ref_count(ecl_file_kw_type * file_kw) {
```

Comment:
This should *not* be a publicly visible function. The use of a reference counter is an implementation detail.